src/gpt_review/repositories/devops.py

```diff
@@ -284,17 +288,20 @@ def _get_changed_blobs(self, pull_request: GitPullRequest) -> List[str]:
 
         skip = 0
         while True:
+            # TODO error handling?
+            # TF401029: Couldn't find Git commit with ID ee3ca002f2e07b3a33321eeb2614a22d7a324bef.
             pr_commits = self.client.get_commit_diffs(
                 repository_id=self.repository_id,
                 project=self.project,
                 diff_common_commit=False,
                 base_version_descriptor=GitBaseVersionDescriptor(
-                    base_version=pull_request["lastMergeSourceCommit"]["commitId"], base_version_type="commit"
+                    base_version=pull_request.last_merge_source_commit.commit_id, base_version_type="commit"
```

Comment:
Here, I think we need to check if pull_request is type GitPullRequest or is type dict to make sure it works correctly in both ways.

Or, we can try and track down where in the code pull_request is getting used as a dict, and try to convert it to a GitPullRequest (but that may be harder)

Reply:
O looks like we may be already doing that `pull_request = GitPullRequest(pull_request_event["pullRequest"])`, this may be all set than.